float64 time_moment
---
bool lost_marker
